wagtailio/navigation/models.py

```diff
@@ -0,0 +1,80 @@
+from django.db import models
+
+from modelcluster.models import ClusterableModel
+from wagtail.admin.edit_handlers import FieldPanel, StreamFieldPanel
+from wagtail.contrib.settings.models import BaseSetting, register_setting
+from wagtail.core.fields import StreamField
+from wagtail.snippets.edit_handlers import SnippetChooserPanel
+from wagtail.snippets.models import register_snippet
+
+from wagtailio.navigation.blocks import MainMenuSectionBlock, NavStreamField
+
+
+@register_snippet
+class FooterMenu(models.Model):
+    name = models.CharField(max_length=255)
+    sections = StreamField(
+        NavStreamField(),
+    )
+
+    panels = [
+        FieldPanel("name"),
+        StreamFieldPanel("sections"),
+    ]
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        verbose_name = "Footer menu"
+
+
+@register_snippet
+class MainMenu(ClusterableModel):
```

Comment:
Now that we're not using snippets for the sections, this could just be a `Model` again. Not a dealbreaker, but simpler is betterer :smile:

Reply:
`MainMenu` is still a snippet